com.rocksandpaper.syndic.yml

```
@@ -0,0 +1,23 @@
+# SPDX-FileCopyrightText: 2022 Connor Carney <hello@connorcarney.com>
+# SPDX-License-Identifier: GPL-3.0-or-later
+id: com.rocksandpaper.syndic
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: syndic
+rename-icon: syndic
+finish-args:
+- --share=network
+- --share=ipc
+- --socket=x11
```

```suggestion
- --socket=fallback-x11
```

if it actually support wayland.